### PR TITLE
GhostBSD: remove old TrueOS files

### PR DESCRIPTION
**Pull request by HacKurx**
Unless I am mistaken, TrueOS is dead.

### CODE REVIEW

**Comment by HacKurx** [comment]
[cleaning_of_2114_files.txt](https://github.com/ghostbsd/ghostbsd-ports/files/6372055/cleaning_of_2114_files.txt)

**Comment by vimanuelt** [comment]
Yes, you are mistaken. GhostBSD is the successor of these files.

**Comment by HacKurx** [comment]
However, the patch allows you to see that 1304 URLs do not work anymore.
[URLs_deads.txt](https://github.com/ghostbsd/ghostbsd-ports/files/6372117/URLs_deads.txt)

**Comment by HacKurx** [comment]
To avoid this kind of misunderstanding, please rename the extensions ".trueos" to ".ghostbsd" :wink:

File: net/xorp/Makefile.trueos [hunk 1]
```diff
@@ -1 +0,0 @@
-USE_OPENRC_SUBR=	openrc-xorp
```
Comment (ericbsd) [inline net/xorp/Makefile.trueos:1]: Removing that file would broke openrc for this packages

Comment (ericbsd) [reply net/xorp/Makefile.trueos:1]: There is a lot Makefile.trueos whit USE_OPENRC_SUBR and we need those for openrc.

**Comment by HacKurx** [comment]
Ok thanks to both of you. I was just passing through here because I was comparing some of your ports. 

I will close this pull requests.

Thank you.

**Comment by ericbsd** [comment]
> Ok thanks to both of you. I was just passing through here because I was comparing some of your ports.

@HacKurx Comparing with what?

**Comment by HacKurx** [comment]
> > Ok thanks to both of you. I was just passing through here because I was comparing some of your ports.
> 
> @HacKurx Comparing with what?

@ericbsd [hardenedbsd](https://git.hardenedbsd.org/hardenedbsd/ports/-/commit/d0bbddebe6afe9c6329668216d2e42b90505cfaf).

**Comment by ericbsd** [comment]
> > > Ok thanks to both of you. I was just passing through here because I was comparing some of your ports.
> > 
> > 
> > @HacKurx Comparing with what?
> 
> @ericbsd [hardenedbsd](https://git.hardenedbsd.org/hardenedbsd/ports/-/commit/d0bbddebe6afe9c6329668216d2e42b90505cfaf).

The problem is that GhostBSD is not using the same init system. Also Makefile.trueos was created to avoid merge conflict with upstream Makefile. Makefile.trueos that only contain URL can be removed, but all the Makefile.trueos that contain USE_OPENRC_SUBR need to stay in.

**Pull request closed without merging**